jenkins-schema.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+./jenkins.sh
```

Comment:
@rboulton are you referring to this line? https://github.com/alphagov/collections/blob/master/jenkins.sh#L13 I believe that sets the build status for the collections commit, not the govuk-content-schemas commit.